src/py21cmfast/src/UsefulFunctions.c

```diff
@@ -593,6 +593,107 @@ void print_corners_real(float *x, int size){
     printf("\n");
 }
 
+void debugSummarizeBox(float *box, int size, char *indent){
```

Comment:
👍🏻  -- Although, it was actually the `InitialConditions` summarizer that was giving the last segfault, because it was trying to print out 2LPT boxes that didn't exist. It was only happening for `SUPER_DEBUG` which is why I wasn't catching it locally. Super annoying!